_data/guides.yml

```diff
@@ -219,10 +219,12 @@ toc:
         path: /docs/getting-started-guides/fedora/flannel_multi_node_cluster/
       - title: CentOS
         path: /docs/getting-started-guides/centos/centos_manual_config/
-      - title: Ubuntu
-        path: /docs/getting-started-guides/ubuntu/
+      - title: CoreOS
+        path: /docs/getting-started-guides/coreos
       - title: CoreOS with Calico
         path: /docs/getting-started-guides/coreos/bare_metal_calico/
+      - title: Ubuntu
```

Comment:
Confusing to have CoreOS link right between two Ubuntu links. Opting to keep OS's together.